src/views/GroupTable/GroupTable.tsx

```diff
@@ -58,10 +58,18 @@ const GroupTable: FC<Props> = props => {
 				dispatch(searchGroupTable(text))
 			}} />
 			<div className={classes.content}>
-				<UsersTable users={filteredUsersTable}
-							onUserClicked={(predictionId: number) => {
-								history.push(`/prediction/${predictionId}`);
-							}} />
+				{
+					groups.fetching ?
+					<div className={classes.center}>
+						<CircularProgress/>
+					</div>
+					:
```

Comment:
one you will merge the previous PR - this will be in conflict with what you already did :)